modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/bbcode/bbcode_parser.js

```diff
@@ -628,7 +628,10 @@
 
 			const imageSrcInput = instance._extractData(STR_IMG, true);
 
-			if (REGEX_IMAGE_SRC.test(imageSrcInput)) {
+			if (
+				REGEX_IMAGE_SRC.test(imageSrcInput) ||
+				imageSrcInput.startsWith('data:image/')
```

Comment:
Can you reverse the test? `startsWith` should be much faster than a regex test.
```
if (imageSrcInput.startsWith(...) || REGEX_IMAGE_SRC.test(...))
```

Reply:
i'll do it in both cases :)

Reply:
yes, I'll do it in both cases